VERSION

```diff
@@ -41,7 +41,7 @@ flex_min_version=2.5.4
 # requirement is that it must be entirely printable ASCII characters
 # and have no white space.
 
-greek=rc1
+greek=a1
```

Comment:
Don't you want to make this `rc2`?

We've already made `a1` releases -- don't go backwards.

You can make a `rc2` release tarball, and if it passes a few sanity tests, make a final tarball (without `rc2`).  The Jenkins job has a parameter that will set `greek` to empty during the tarball build (i.e., you don't have to commit `greek=` in `VERSION` to make this happen).

![Screenshot 2023-12-20 at 3 13 39 PM](https://github.com/open-mpi/ompi/assets/617316/a8285a25-005b-41c5-b147-8f53cbda7d81)

Reply:
Ah you are right. We should only go forward.

Updated. Thanks!